### PR TITLE
Update scripts and remove unnecessary stuff from main-source

### PR DESCRIPTION
**Pull request by aiyengar2**
This PR is part of a migration to a new structure for this Github repository.

After the necessary PRs are merged, changes to the chart will be made to the `main-source` branch and, on successful push, the assets that are generated from it will be automatically committed to the `main` branch.

Related PR: https://github.com/rancher/partner-charts/pull/40

### CODE REVIEW

File: .github/workflows/pull-request.yaml [hunk 1]
```diff
@@ -2,7 +2,8 @@ name: CI-pullrequest
 
 on:
   pull_request:
-    branches: [ main ]
+    branches:
+      - dev-v2.5-source
```
Comment (StrongMonkey) [inline .github/workflows/pull-request.yaml:6]: main-source?

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -2,7 +2,8 @@ name: CI-push
 
 on:
   push:
-    branches: [ main ]
+    branches:
+      - dev-v2.5-source
```
Comment (StrongMonkey) [inline .github/workflows/push.yml:6]: same

File: .github/workflows/pull-request.yaml (hunk 1 above)
Comment (StrongMonkey) [reply .github/workflows/pull-request.yaml:6]: nvm, saw your comments

**Comment by StrongMonkey** [comment]
LGTM, make sure you fixed the CI.

**Comment by aiyengar2** [comment]
Copies over changes from currently unmerged https://github.com/rancher/charts/pull/688 as well to fix CI

**Review by cbron — APPROVED** [review]
Didn't review the scripts obviously, but delete/adds look about right.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# partner-charts
+## partner-charts
```
[the remark this answers is not in the document]
Comment (aiyengar2) [reply README.md:1]: cc: @PennyScissors if you have a PR out for the rewrite

**Pull request merged**